### ISSUE TITLE
Font support for East Asian languages.

### ISSUE THREAD

**Issue by oo13** [naev/naev#1896]
Supporting for East Asian languages we need a mechanism to change some fonts for each language. I suggest letting the translators specify the font set.

Unicode put the different "glyphs" in Chinese and Japanese into a single code-point (cf. [Unicode FAQ Chinese and Japanese #2](https://www.unicode.org/faq/han_cjk.html#2), [Unicode FAQ Chinese and Japanese #3](https://www.unicode.org/faq/han_cjk.html#3)), that is, there is no universal font in Unicode.

By the way, the current main seem to have Korean fonts, but they don't fully support either Japanese or Chinese.

**Comment by bobbens** [comment]
Thank you for the issue. That should actually be possible already when creating a new translation. In particular, the following strings are translatable:
```
#: src/ndata.h:61
msgid "Cabin-SemiBold.otf,NanumBarunGothicBold.ttf,SourceCodePro-Semibold.ttf"
msgstr ""

#. *< Default font path.
#: src/ndata.h:62
msgid "SourceCodePro-Semibold.ttf,D2CodingBold.ttf"
msgstr ""
```
This specifies the order of the fonts and their respective fallbacks. Each translated language is able to choose what fonts to use and what order to use them in. Also, whenever other fonts are used in-game, they also generate a translatable string for the font name allowing that to be specified in a translation. New fonts can be added as necessary.

**Comment by oo13** [comment]
Thanks, but it's not working.

1. I created [ja.po](https://github.com/naev/naev/files/7013928/ja.po.txt).
2. added "ja" to LINGUAS.
3. put ipaexg.ttf to naev/artwork.fonts.
4. cd build && ./naev.sh
5. Naev shows almost the scrolled text fine but not "々". I suspect Naev uses D2CodingBold.ttf.
6. ./naev.sh outputs [the log](https://github.com/naev/naev/files/7013931/log.txt).

Naev version: 4b5fb5ed99b671e918553e499c62e4479703e992
OS: Debian GNU/Linux Testing

You can download ipaexg.ttf from https://moji.or.jp/ipafont/ipaex00401/. It might not have English page, sorry.

**Comment by bobbens** [comment]
OK, we were missing translating the font name in a few places (including the console), they should be fixed in cc2012e10f2cfaae80c84a4d063f82c204b49975 . Thanks for the report! Please re-open if I missed something.
